content/en/news/releases/1.4.x/announcing-1.4.10/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Announcing Istio 1.4.10
+linktitle: 1.4.10
+subtitle: Patch Release
+description: Istio 1.4.10 security release.
+publishdate: 2020-06-18
+release: 1.4.10
+aliases:
+    - /news/announcing-1.4.10
+test: n/a
```

Comment:
I don't think the test: tag is needed under en/news. Probably doesn't hurt. @frankbu Will we ever have tests outside of docs?

Reply:
This came from the announcement for 1.5.5, but I changed the version numbers.

Reply:
OK. I noticed Francois had it as well. I'll ask in today's WG call about where this should be used.